sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/splittabledofn/ByteKeyRangeTrackerTest.java

```diff
@@ -287,4 +318,30 @@ public void testBacklogPartiallyCompleted() {
     tracker.tryClaim(ByteKey.of(0xa0));
     assertThat(tracker.getSize(), allOf(greaterThan(0.), lessThan(1.)));
   }
+
+  /** Asserts the two ByteKey are equal except trailing zeros. */
+  private static void assertByteKeyEqualExceptPadding(ByteKey expected, ByteKey key) {
```

Comment:
This is taken from https://github.com/apache/beam/blob/master/sdks/java/core/src/test/java/org/apache/beam/sdk/io/range/ByteKeyRangeTest.java#L365-L382

Reply:
Make that method public and call it from this test since they are both in the same module.